File: TransCoder_tokenized_test_set_functions/python/SEARCH_INSERT_AND_DELETE_IN_A_SORTED_ARRAY.py
def binarySearch ( arr, low, high, key ) :
    if ( high < low ) :
        return - 1
    mid = ( low + high ) / 2
    if ( key == arr [ int ( mid ) ] ) :
        return mid
    if ( key > arr [ int ( mid ) ] ) :
        return binarySearch ( arr, ( mid + 1 ), high, key )
    return ( binarySearch ( arr, low, ( mid - 1 ), key ) )

File: results/java/SEARCH_INSERT_AND_DELETE_IN_A_SORTED_ARRAY.cpp
int f_filled ( int arr [ ] , int low , int high , int key ) {
  if ( high < low ) return - 1 ;
  int mid = ( low + high ) / 2 ;
  if ( key == arr [ mid ] ) return mid ;
  if ( key > arr [ mid ] ) return f_filled ( arr , ( mid + 1 ) , high , key ) ;
  return f_filled ( arr , low , ( mid - 1 ) , key ) ;
}
